spec/factories.rb

```diff
@@ -6,4 +6,8 @@
     email Faker::Internet.email
     password '369852'
   end
+  factory :news do
```

Comment:
Each model should have its own factory file.

Reply:
split this file into two

Reply:
one file for users  and another for news?

Reply:
yes